examples/imagefilter/node/package.json

```
@@ -0,0 +1,17 @@
+{
+    "name": "imagefilter",
+    "version": "1.3.0",
+    "main": "main.js",
```

Needed for top-level await

```suggestion
    "main": "main.js",
    "type": "module",
```